pkg/dogstatsd/server_test.go

```diff
@@ -244,3 +244,45 @@ func TestUDPForward(t *testing.T) {
 
 	assert.Equal(t, message, buffer)
 }
+
+func TestHistToDist(t *testing.T) {
+	port, err := getAvailableUDPPort()
+	require.NoError(t, err)
+	config.Datadog.SetDefault("dogstatsd_port", port)
```

Comment:
You should set back the default value in a `defer` to not impact following test.